src/analyses/variable-sensitivity/variable_sensitivity_domain.h

```diff
@@ -0,0 +1,184 @@
+/*******************************************************************\
+
+ Module: analyses variable-sensitivity
+
+ Author: Thomas Kiley, thomas.kiley@diffblue.com
+
+There are different ways of handling arrays, structures, unions and
+pointers.  Our existing solution basically ignores them which is
```

Comment:
Concretely refer to the module/class of the existing solution.

Reply:
Done.